5_databases/week2/part3GetallThe taskAssigned to user.sql

```diff
@@ -0,0 +1,57 @@
+use hyf_database
+
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+SELECT*
```

Comment:
So I tried your queries locally on the same database attached in the homework repo none of them works. have you tried them?

Reply:
Hey Mr Younes< i Hope you are fine,
 they worked in my pc.